### PR TITLE
[pipeline] fix return_dict & fix pure_pipeline_test

### PR DESCRIPTION
**Pull request by Fridge003**
## 📌 Checklist before creating the PR

- [ ] I have created an issue for this PR for traceability
- [x] The title follows the standard format: `[doc/gemini/tensor/...]: A concise description`
- [ ] I have added relevant tags if possible for us to better distinguish different PRs

## 🚨 Issue number

> Link this PR to your issue with words like fixed to automatically close the linked issue upon merge
>
> e.g. `fixed #1234`, `closed #1234`, `resolved #1234`

## 📝 What does this PR do?

> Summarize your work here.
> if you have any plots/diagrams/screenshots/tables, please attach them here.

- fix the misconfiguration of setting return_dict to False in bert.py and bloom.py under shardformer/modeling
- fix import warning in opt.py
- fix test_pure_pipeline test in test_shardformer

## 💥 Checklist before requesting a review

- [ ] I have linked my PR to an issue ([instruction](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue))
- [x] My issue clearly describes the problem/feature/proposal, with diagrams/charts/table/code if possible
- [x] I have performed a self-review of my code
- [x] I have added thorough tests.
- [x] I have added docstrings for all the functions/methods I implemented

## ⭐️ Do you enjoy contributing to Colossal-AI?

- [x] 🌝 Yes, I do.
- [ ] 🌚 No, I don't.

Tell us more if you don't enjoy contributing to Colossal-AI.

### CODE REVIEW

**Comment by Fridge003** [comment]
截屏2023-07-26 15 00 38

**Review by CjhHa1 — APPROVED** [review]
LGTM

**Review by CjhHa1 — APPROVED** [review]

**Review by CjhHa1 — APPROVED** [review]
pass

File: colossalai/shardformer/modeling/bert.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import warnings
```
Comment (CjhHa1) [inline colossalai/shardformer/modeling/bert.py:1]: lgtm

**Review by CjhHa1 — APPROVED** [review]

**Review by CjhHa1 — APPROVED** [review]

**Review by ver217 — APPROVED** [review]

**Pull request merged**